vocode/streaming/transcriber/azure_transcriber.py

```diff
@@ -74,54 +74,60 @@ def __init__(
         else:
             speech_params["language"] = self.transcriber_config.language
 
-        if transcriber_config.stable_partial_result_threshold:
+        if self.transcriber_config.stable_partial_result_threshold:
             speech_config.set_property(
                 property_id=speechsdk.PropertyId.SpeechServiceResponse_StablePartialResultThreshold,  # pylint: disable=line-too-long
                 value=str(self.transcriber_config.stable_partial_result_threshold),
             )
 
-        if transcriber_config.segmentation_sil_timeout:
+        if self.transcriber_config.segmentation_sil_timeout:
             speech_config.set_property(
                 property_id=speechsdk.PropertyId.Speech_SegmentationSilenceTimeoutMs,
                 value=str(self.transcriber_config.segmentation_sil_timeout),
             )
 
-        if transcriber_config.end_sil_timeout:
+        if self.transcriber_config.end_sil_timeout:
             speech_config.set_property(
                 property_id=speechsdk.PropertyId.SpeechServiceConnection_EndSilenceTimeoutMs,  # pylint: disable=line-too-long
                 value=str(self.transcriber_config.end_sil_timeout),
             )
 
-        if transcriber_config.enable_dictation:
+        if self.transcriber_config.enable_dictation:
             speech_config.enable_dictation()
 
         self.speech = speechsdk.SpeechRecognizer(**speech_params)
 
-        if transcriber_config.phrases:
+        if self.transcriber_config.phrases:
             speech_phrase_list_grammer = speechsdk.PhraseListGrammar.from_recognizer(
                 self.speech
             )
-            for phrase in transcriber_config.phrases:
+            for phrase in self.transcriber_config.phrases:
                 speech_phrase_list_grammer.addPhrase(phrase)
 
         self._ended = False
         self.is_ready = False
 
     def recognized_sentence_final(self, evt):
+        start = evt.result.offset / 1e7
+        duration = evt.result.duration / 1e7
```

Comment:
where's the `1e7` come from?

Reply:
https://learn.microsoft.com/en-us/azure/ai-services/speech-service/get-speech-recognition-results?pivots=programming-language-python

unit is 100 nano seconds